From 5905e4e5e8add2ffba8dd7486b941ba61e0a2c4f Mon Sep 17 00:00:00 2001
From: Richard Zhao <b20223@freescale.com>
Date: Tue, 19 Aug 2008 16:13:14 +0800
Subject: [PATCH] ENGR00087647 Fix SSI underrun

Alloc DMA buffer in DDR rather than in IRAM, because IRAM size is too small.

Signed-off-by: Richard Zhao <b20223@freescale.com>
---
 arch/arm/configs/imx37_3stack_defconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/configs/imx37_3stack_defconfig b/arch/arm/configs/imx37_3stack_defconfig
index e01693a..fb08ff1 100644
--- a/arch/arm/configs/imx37_3stack_defconfig
+++ b/arch/arm/configs/imx37_3stack_defconfig
@@ -1038,7 +1038,7 @@ CONFIG_SND_SOC=y
 #
 CONFIG_SND_MXC_SOC=y
 CONFIG_SND_MXC_SOC_SSI=y
-CONFIG_SND_MXC_SOC_IRAM=y
+# CONFIG_SND_MXC_SOC_IRAM is not set
 CONFIG_SND_SOC_IMX_3STACK_WM8350=y
 # CONFIG_SND_SOC_IMX_3STACK_AK4647 is not set
 # CONFIG_SND_SOC_IMX_3STACK_WM8580 is not set
-- 
1.5.4.4

